datafusion/physical-expr-common/src/aggregate/stats.rs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
let me take a to-do note. Before all other functions are moved to `functions-aggregate` we need it to keep in `common`, since we don't import physical-expr into `functions-aggregate`